presto-main/src/main/java/com/facebook/presto/operator/PagesSpatialIndexFactory.java

```diff
@@ -69,13 +69,11 @@
 
     @GuardedBy("this")
     private final List<SettableFuture<PagesSpatialIndex>> pagesSpatialIndexFutures = new ArrayList<>();
-
+    private final ReferenceCount activeProbeOperators = new ReferenceCount(1);
```

Comment:
perhaps, add an empty line after this to visually separate final variables from mutable ones.